### DISCUSSION TITLE
Legion 5 17ACH6

### DISCUSSION THREAD

**Discussion by CasanovaSan** [kalkmann/Legion-5600H-Hackintosh#9] [Q&A]
is it possible for my laptop to do this hackintosh?

**Comment by kalkmann** [comment]
> is it possible for my laptop to do this hackintosh?

You can try with https://github.com/NootInc/NootedRed - Instructions: https://nootinc.github.io/nred and https://nootinc.github.io/guide/gathering-files/kexts

You can receive help in forum https://forum.amd-osx.com/ and discord https://discord.gg/EfCYAJW
